index.rst

```diff
@@ -27,6 +27,10 @@ The recommended way to install Mink with all its dependencies is through
 
     $ composer require behat/mink
 
+.. note::
+    For local installations of composer you must call it like this: ``$ php composer.phar require behat/mink`` .
```

Comment:
Can you add an empty line between the ``.. note::`` directive and its content and wrap the content after the first word crossing 72 chars to keep the lines shorter in the source ?

@aik099 FYI, this CS rule about the wrapping comes from the Symfony documentation standards. I decided to apply the same standards